stable/kommander-karma/values.yaml

```diff
@@ -0,0 +1,69 @@
+# Default values for kommander-karma.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Internal address for the cluster's alertmanager.
+# alertmanagerAddress: "HOST:PORT"
+alertmanagerAddress: ""
+
+federate:
+  systemNamespace:
+    name: kommander-system
+
+karma:
+  ingress:
```

Comment:
Would you mind adding
```
service:
  labels:
    servicemonitor.kubeaddons.mesosphere.io/path: "kommander__monitoring__karma__metrics"
```
block under karma? This is so that the metrics endpoint can be discovered by Prometheus